src/BetweenTheBubbles.tbasic

```diff
@@ -22,8 +86,13 @@ function tbMain()
   dword hWin = tbgl_createWindowEx("Between the Bubbles", 720, 720, 32, %TBGL_WS_WINDOWED | %TBGL_WS_DONTSIZE | %TBGL_WS_CLOSEBOX)
   tbgl_showWindow
   
+  tbgl_UseTexturing %TRUE
```

Comment:
Hi Sebastian,

I would recommend to avoid these evil functions changing the global state, and use `TBGL_PushState %TBGL_TEXTURING / TBGL_PopState` on appropriate places.

The same for AlphaTest on the line below.

Reply:
Perfectly agree, but updates will be for [feat_Intro Draft](https://github.com/petrSchreiber/BetweenTheBubbles-tb/pull/13) as this fix branch is doomed to disappear.